INFO     logger:Big_Data_Travel_Card.py:82 本次的问题为：False
INFO     logger:Big_Data_Travel_Card.py:83 回答的答案为：False